tutorials/poststack_nccl.py

```diff
@@ -0,0 +1,197 @@
+r"""
+Post Stack Inversion - 3D with Nvidia's NCCL
+=========================
+This tutorial is similar to poststack.py but it shows how to run PyLops-MPI in multi-GPU environment
+and have those GPUs communicate via NCCL
+"""
+
+import numpy as np
+import cupy as cp
+from scipy.signal import filtfilt
+from matplotlib import pyplot as plt
+from mpi4py import MPI
+
+from pylops.utils.wavelets import ricker
+from pylops.basicoperators import Transpose
+from pylops.avo.poststack import PoststackLinearModelling
+
+import pylops_mpi
+import pylops_mpi.utils
+import pylops_mpi.utils._nccl
+
+plt.close("all")
+rank = MPI.COMM_WORLD.Get_rank()
+# size = MPI.COMM_WORLD.Get_size()
```

Comment:
Remove if not used... I just realized it is also not used in the original tutorial...